### PR TITLE
Lint throw message format

### PR DESCRIPTION
**Pull request by haya14busa**
ref: #470

### CODE REVIEW

**Comment by haya14busa** [comment]
I added linter for throw message format in vital.vim.
In vital.vim, throw messages should be start with `vital: {module-name}: `.

I feel it's over-killing to create a repository just for vital.vim specific linter and prepare linter binary, so I added a single golang script under scripts directory for now. We can make it better later when adding more vital.vim specific checks.

**Comment by haya14busa** [comment]
related docker image change: https://github.com/haya14busa/dockerfile-lint-vital.vim/commit/8b45f0d1d5260ca7131ab6c8e409b771814f53dd

**Comment by thinca** [comment]
しゅごい…。
Approve しましたが、コミットの整理はしてもらって大丈夫です。go ファイルの修正を 1 つにまとめたりなど。

File: reviewdog.yml [hunk 1]
```diff
@@ -13,3 +13,8 @@ runner:
       - "%f:%l:%c:%trror: %m"
       - "%f:%l:%c:%tarning: %m"
       - "%f:%l:%c:%m"
+
+  vital-throw-message:
+    cmd: go run ./scripts/lint-throw.go $(find autoload/vital/__vital__/ | grep -e '.vim$')
```
Comment (rhysd) [inline reviewdog.yml]: nit: `\.vim$`

Comment (haya14busa) [inline reviewdog.yml]: Thanks. Nice catch! :baseball:  :dog2:

File: scripts/lint-throw.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	"os"
+	"strings"
+
+	vimlparser "github.com/haya14busa/go-vimlparser"
+	"github.com/haya14busa/go-vimlparser/ast"
+	"github.com/haya14busa/go-vimlparser/token"
+)
+
+func main() {
+	flag.Parse()
```
Comment (rhysd) [inline scripts/lint-throw.go]: 良ければ簡単な使い方を `-h` オプションとかで書いておいてもらえるとローカルで使いたい時助かりそうです

Comment (haya14busa) [inline scripts/lint-throw.go]: Please take another look https://github.com/vim-jp/vital.vim/pull/471/commits/17a0aac9ad648979f10cc7315abfd4945c17646a

**Comment by haya14busa** [comment]
Thank you for the merge :bird: :sparkles:

**Pull request merged**